### ISSUE TITLE
Extend Courthouse features

### ISSUE THREAD

**Issue by SunlightKim** [SnailyCAD/snaily-cadv4#833]
### Describe the feature

It would be really nice to extend the courthouse section of the CAD.
For example, it would be cool to be able to have something for lawyers, a "bar exam" (the exam for lawyers), etc.
I hope you know what I mean and maybe you also have some ideas.

I stay available if you have any questions.

**Comment by Dev-CasperTheGhost** [comment]
I actually don't understand 😅! I'm not used to these kind of interactions IRL, I'm going to need some more information. 

What would you actually like to see? Not just "an extend to the courthouse"

**Comment by SunlightKim** [comment]
> I actually don't understand 😅! I'm not used to these kind of interactions IRL, I'm going to need some more information.
> 
> What would you actually like to see? Not just "an extend to the courthouse"

Oh no problem, don't worry!
Basically, in the courthouse, it would be quite nice to have a section for lawyers, a bit like the LEO/EMS section, you know?
This section would of course be whitelisted.
Another good thing would be to add in the courthose section a page with the laws listed (similar to the Penal Code in LEO but open to public), as well as some other informations.

To make it a bit simpler, just a page in the courthouse, that everyone can see, where we can post the laws, government informations, etc.

Is it a bit clearer?

**Comment by Dev-CasperTheGhost** [comment]
> Another good thing would be to add in the courthose section a page with the laws listed (similar to the Penal Code in LEO but open to public), as well as some other informations.

> To make it a bit simpler, just a page in the courthouse, that everyone can see, where we can post the laws, government informations, etc.

Who would be able to manage this information? Maybe a `ManageCourthouseInfo` permission?

**Comment by SunlightKim** [comment]
> > Another good thing would be to add in the courthose section a page with the laws listed (similar to the Penal Code in LEO but open to public), as well as some other informations.
> > To make it a bit simpler, just a page in the courthouse, that everyone can see, where we can post the laws, government informations, etc.
> 
> Who would be able to manage this information? Maybe a `ManageCourthouseInfo` permission?

Yup, a ManageCourthouseInfo perm would be good, since it would be "official/governement info", we wouldn't want random people posting random things haha

**Comment by SunlightKim** [comment]
> > Another good thing would be to add in the courthose section a page with the laws listed (similar to the Penal Code in LEO but open to public), as well as some other informations.
> > To make it a bit simpler, just a page in the courthouse, that everyone can see, where we can post the laws, government informations, etc.
> 
> Where/how would someone add these information posts?

Well, since there is a "Manage courthouse" page in the Admin section, it seems good to do it there.
![image](https://user-images.githubusercontent.com/104458119/171416518-2c3c5b71-6d73-444c-9172-21ea233053a6.png)
Just like there are currently 3 pages in the courthouse, you could add a fourth one named, let's say "Information Posts".
There'd be a "Create post" for people who have the ManageCourtInfo permission.
Basically, a bit like Bleeter, but for the courthouse.

**Comment by SunlightKim** [comment]
> > Just like there are currently 3 pages in the courthouse, you could add a fourth one named, let's say "Information Posts".
> > There'd be a "Create post" for people who have the ManageCourtInfo permission.
> > Basically, a bit like Bleeter, but for the courthouse.
> 
> These posts are **visible** to anyone? And would the posts be directly displayed, or would the post need to be opened in a modal/page?

Posts would be visible to anyone, yes and they should be opened in a modal/page, that would be the best.
